### DISCUSSION TITLE
beta1

### DISCUSSION THREAD

**Discussion by akorotkov** [orioledb/orioledb#249] [Announcements]
The first beta release of OrioleDB incorporates the following changes.

fe97326 Stamp beta 1
b4f8162 issue 240 fix
8c4a501 Implement parallel index build in recovery
081feeb Parallel index build
778a621 Change file structre
317ca40 Update copyright
81e98d3 Index build concurrent checkpoint test
d7dc1bf issue 242 fix
694767a issue 241 fix
4269f5e issue 223 fixed
44f682c issue 212 comment fix
49b7f7b issue 232 fix
a5f3639 issue 218 fix
8adb8d0 issue 215 fix
aedbad6 issue 239 fix
608b840 Remove some redundencies in recovery
f52f774 Enchance benchmark task
bb5cc26 Update the aws credentials action
ab48a20 Update security group
d6b915a Configure compiler and version
aa170ec Don't use asserts during benchmarking
232a434 EC2 worklow
9d5f9d5 Don't do 'make install' for static check type
27e242e Turn back silent build of PostgreSQL sources
f1a8375 spelling mistake
81c7912 as parallel as we can afford to be
9bfec31 minor changes to the CI build script
f282011 Improve Dockerfile: llvm16 workarounds + doc + local test (#243)
828e94b Make testgres directories depending on test name
e2780c5 Check for duplicate update in INSERT .. ON CONFLICT UPDATE ...
8c47638 Fix valgrind complaint
04e35a9 Partial fix for parallel mode
ea529a0 Improve error message
0c478ca Fix issue 229
adad990 Changed timestamp (yech!) to timestamptz
e7c0ecc Remove unused struct
5657366 pgindent run
1c2fd3d pgindent CI task
5af1b74 Rework index build concurrency
686928a Another fix for force_parallel_mode
0132ed3 pgindent
58eade7 Rename oTablesAddLock => oTablesMetaLock
2a0ee77 Fix usage of force_parallel_mode another time
460ee3f Improve error messages
8726dd4 issue 138 test and error
401d82c issue 47 test, errors and docs
6dc4f7b issue 46 test, errors and docs
62704af issue 44 test, errors and docs
3f26305 issue 212 fix
28bd391 issue 211 fix
f0e574b issue 209 fix
3a96355 issue 221 fix
2275c90 issue 222 fix
f4b79c0 issues 213,214,220 fixes
9802b09 issue 225 fix
787fb7d Fix unnecessary recreate_table_descr_by_oids() on system tree update
2537998 Fix releasing oTablesAddLock
ee4d289 Fix regex to determine the required patchset version
32aa8e8 Fix usage of force_parallel_mode GUC in PG 16
6768130 Invalidate descrs on relation truncate in recovery
4ab2685 Fix persistence issues
d6a4359 Fix some gcc-12 warnings

<hr /><em>This discussion was created from the release <a href='https://github.com/orioledb/orioledb/releases/tag/beta1'>beta1</a>.</em>

**Comment by HiperMaximus** [comment]
Just wanted to say thank you! This is probably one of the most important work being done in DB space. I hope it can become part of PG

**Comment by akorotkov** [comment]
@HiperMaximus, thank you for lovely feedback! Please, check if you can try OrioleDB on your workload or contribute the other way.
